data-prepper-plugins/opensearch/src/main/java/org/opensearch/dataprepper/plugins/sink/opensearch/index/IndexType.java

```diff
@@ -13,9 +13,12 @@
 
 public enum IndexType {
     TRACE_ANALYTICS_RAW("trace-analytics-raw"),
+    TRACE_ANALYTICS_RAW_STANDARD("trace-analytics-standard-raw"),
```

Comment:
The term `standard` is unclear. Can you give a more precise adjective? Perhaps `otel`?

Reply:
We can resolve in a follow-on PR.